nsf/randomized_approaches.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
+title: Randomized Approaches in Scientific Computing
+permalink: /nsf/randomized_approaches.md
```

Comment:
Cannot end in .md. Also, so our addresses are consistent, change this to `/nsf/year3/randomized_approaches`